data/data/azure/bootstrap/variables.tf

```diff
@@ -63,3 +63,9 @@ variable "nsg_name" {
   type        = string
   description = "The network security group for the subnet."
 }
+
+variable "preexisting_network" {
```

Comment:
I think we can drop this as NSG is always created.

Reply:
Fixed